python/whylogs/experimental/extras/nlp_metric.py

```diff
@@ -1,8 +1,11 @@
 from copy import deepcopy
 from dataclasses import dataclass, field
 from itertools import chain
+from tempfile import NamedTemporaryFile
 from typing import Any, Dict, List, Optional, Tuple, Union
 
+from pybloomfilter import BloomFilter
```

Comment:
We're now requiring the `nlp` extra to be installed to anyone who wants to use BagofWords, even if not interested in OOV functionalities (as is the case with some of our previous examples). Just raising the question if we think this is ok, or would we like to import this only when actually using OOV features?

Reply:
Right now, the BoW and SVD metrics are in the same file. SVD requires numpy. I could split BoW into a separate file/extra, but the I don't Bloom filter package is an onerous dependency.